src/App.jsx

```diff
@@ -14,7 +14,7 @@ function App() {
           {/* Homepage*/}
           <Route path="/" element={<InvoiceContainer />} />
           {/* ViewInvoice*/}
-          <Route path="/ViewInvoice" element={<ViewInvoice />} />
+          <Route path="/ViewInvoice/:idName/:id" element={<ViewInvoice />} />
```

Comment:
![](https://i.makeagif.com/media/5-19-2015/ZGns0a.gif)

Why have both IDs? The API just needs the id, so go with that!